Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocColumnRendererArrayBuilder.cs

```diff
@@ -201,17 +198,11 @@ private bool IsColumnVisibleForBocValidationErrorIndicatorColumnDefinition (BocV
 
     private bool IsColumnVisibleForBocCommandColumnDefinition (BocCommandColumnDefinition columnAsCommandColumn)
```

Comment:
As this method now returns a constant and it is private, we can inline the value and remove a bit of code at the usage.